variables.tf

```
@@ -125,7 +125,7 @@ variable "role_policy" {
 }
 
 variable "secrets" {
-  type        = map
+  type        = map(any)
```

Would this not be a better fit perhaps?

```suggestion
  type        = map(string)
```